Clean up and simplify openshift-ansible.spec

sdodson opened the pull request:
https://jira.coreos.com/browse/BYOH-234

mtnbikenc commented on openshift-ansible.spec:
```diff
@@ -79,103 +55,8 @@ cp -rp roles %{buildroot}%{_datadir}/ansible/%{name}/
-# ----------------------------------------------------------------------------------
-# openshift-ansible-tests subpackage
-# ----------------------------------------------------------------------------------
-%package test
```
CI was updated to install this test package.  Should we do something different there?
https://github.com/openshift/openshift-ansible/blob/master/images/installer/Dockerfile#L30

sdodson replied:
I'll add it back as a sub-package. I don't think we'd want to ship it in our channels though.

mtnbikenc replied:
I agree we should not ship openshift-ansible-test.

mtnbikenc commented on README.md:
```diff
@@ -69,15 +69,14 @@ Requirements:
 
 - Ansible >= 2.7.8
 - Jinja >= 2.7
-- pyOpenSSL
```
README.md is getting an overhaul in the 4.1 Docs PR.  Maybe drop this file from your PR to prevent merge conflicts?

mtnbikenc commented:
/lgtm cancel
Need to drop the `Requires` for the openshift-ansible-test` sub-package.  Image build test is failing.

sdodson commented:
>  Need to drop the `Requires` for the openshift-ansible-test` sub-package. Image build test is failing.

fixed, rebased down to one commit as well.

vrutkovs commented:
/retest

vrutkovs commented:
`release "release-latest" failed: could not create watcher for pod: unknown (get pods)`
/retest

sdodson commented:
/override ci/prow/e2e-aws-scaleup-rhel7

sdodson commented:
/override ci/prow/e2e-aws-scaleup-rhel7

sdodson commented:
/test unit
/test images
/override ci/prow/e2e-aws-scaleup-rhel7

sdodson commented:
/test unit
/test images

sdodson commented:
/test unit
